tox.ini

```diff
@@ -8,7 +8,7 @@ envlist = pypy, py27, py33, py27-twisted-14, py33-twisted-14,
 commands =
     {envpython} setup.py --version
     pip install eliot[dev]
-    coverage run -m unittest discover eliot.tests
+    coverage run -m unittest discover {posargs:eliot.tests}
```

Comment:
Probably something for a follow up issue, but shouldn't we restrict coverage to eliot modules by adding `--source=eliot` and add a `--branches` argument too.